pkg/os/exec.go

```diff
@@ -11,6 +11,7 @@ import (
 
 func runCmd(command string, args []string, env map[string]string) (string, string, error) {
 	cmd := exec.Command(command, args...) // #nosec G204
+	logging.Debugf("Running '%s %s'", command, strings.Join(args, " "))
```

Comment:
nack, `runPrivate` also calls `runCmd`, and we do not want to print the command when `runPrivate` is used.

Reply:
Good catch, will try to change it like we do for oc, send the private flag to run command.